VaticanBAV/comb/BAVcombo24.xml

```diff
@@ -0,0 +1,155 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BAVcombo24" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Lǝssāna sabʾ</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="AD"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0003BAV"/>
+                        <collection>Comboniani</collection>
+                        <idno facs="https://digi.vatlib.it/iiif/MSS_Comb.O.24/manifest.json">Comb. O 24</idno>
+                    </msIdentifier>
+
+                    <msContents>
+                      <summary/>
+
+                      <msItem xml:id="ms_i1" class="content">
+                        <locus from="3r" to="3v" facs="0009">5-6</locus>
+                        <title type="complete" ref="LIT5888AsmatPrayer"/>
+                        <textLang mainLang="gez"/>
+                        <incipit xml:lang="gez">በስመ፡ <gap reason="omitted" extent="unknown"/> ህቡዕ፣ ስሙ፡ ለእግዚአብሔር፡ አስማተ፡ ኃይላት፡ ኢያኤል፡ አፍናኤል፡ አስናኤል፡ አትናኤል፡ ስንኰራኵኤል፡ ፍዳኤል፡ አፍናኤል፡ ኦእግዚእየ፡ ኢየሱስ፡ ክርስቶስ፡ እቀቦ፡ ለግብርከ፡ <del rend="erasure"/></incipit>
```

Comment:
you need `<gap reason="ellipsis">` for abbreviations on the part of the editor (I assume Raineri simply skips the trinitarian formula)
https://betamasaheft.eu/Guidelines/?q=omitted&id=text-encoding

Reply:
Okay. Other examples on BetMas had "omitted", so I have been using that. I'll change the other BAV records where I had "omitted", then.